libpod/runtime_pod.go

```diff
@@ -93,3 +96,41 @@ func (r *Runtime) Pods(filters ...PodFilter) ([]*Pod, error) {
 
 	return podsFiltered, nil
 }
+
+// GetAllPods retrieves all pods
+func (r *Runtime) GetAllPods() ([]*Pod, error) {
+	r.lock.RLock()
+	defer r.lock.RUnlock()
+
+	if !r.valid {
+		return nil, ErrRuntimeStopped
+	}
+
+	pods, err := r.state.AllPods()
+	if err != nil {
+		return nil, err
+	}
+
+	return pods, nil
```

Comment:
return r.state.AllPods() ?

Reply:
@haircommander Fix this and we can merge.